avalon/lib.py

```diff
@@ -98,11 +98,12 @@ def dict_format(original, **kwargs):
         return new_list
 
 
-def which(program):
+def which(program, paths=None):
```

Comment:
Should this get the same changes as we had in `acre`? Or should we remove this change? Currently it has the implementation in here that we didn't like for `acre` 

The downside of duplicate code - hehe.

Reply:
OK I didn't see this comment for three months? I think for compatibility sake with other studios we should keep this. It would effect the approach of launching applications through the launcher.